basic/src/test/java/org/jboss/hal/testsuite/test/configuration/ee/ScheduledExecutorTestCase.java

```diff
@@ -151,18 +152,19 @@ public void addExecutorInGUI() {
         Editor editor = wizard.getEditor();
         editor.text("name", name);
         editor.text(JNDI_NAME, JNDI_VALID);
-        boolean result = wizard.finish();
+        wizard.clickSave();
+        Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent();
```

Comment:
In meantime https://github.com/zebrik/testsuite/commit/4e46b07e7ca2d8543f26915d9b9bb92d3f7b87f6 was merged so you can use `WizardWindow.finishAndDismissReloadRequiredWindow()` instead.